### ISSUE TITLE
Accessibility Violation: The ARIA attributes "aria-checked" are not valid for the element <input> with implicit ARIA role "checkbox

### ISSUE THREAD

**Issue by AmirHussain93** [carbon-design-system/carbon-components-angular#2508]
## Detailed description
This issue is specific to the `<input>` element which is used inside `<ibm-toggle>` component

Some HTML elements or attributes have native semantics that map, by default, to implied ARIA semantics. For predictable interaction and behaviour of UI controls, these implicit semantics should only be overridden as defined in the ARIA in HTML Document conformance requirements for use of ARIA attributes in HTML specification.

If a native HTML attribute fits the intended behaviour, use that instead of adding ARIA markup; Or, update the attributes on the element using the guidance in the Document conformance requirements for use of "ARIA" attributes in HTML section of the ARIA in HTML specification. This reference provides details of valid ARIA markup on native HTML elements.

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
`<ibm-toggle>`

> What did you expect to happen? What happened instead? What would you like to see changed?
ARIA attributes must be valid for the element and ARIA role to which they are assigned

> What browser are you working in?
chrome Version 112.0.5615.49 (Official Build) (arm64)

> What version of the Carbon Design System are you using?
carbon-components@10.5.0
carbon-components-angular@4.71.9

## Additional information

* Screenshots or code
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/27803197/230063641-734bd033-3228-419b-a685-62b233a7981b.png">
